app/Step.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Step extends Model
+{
+    const CREATED_AT = 'created';
+    const UPDATED_AT = 'updated';
+
```

Comment:
No `$fillable` or `$hidden`?

Reply:
Added in the database branch already, which will be merged after this pull request